### PR TITLE
Media caches cleanups

### PR DESCRIPTION
**Pull request by jtkech**
Work in progress.

### CODE REVIEW

File: src/OrchardCore.Modules/OrchardCore.Media/Processing/MediaImageSharpConfiguration.cs [hunk 1]
```diff
@@ -23,8 +23,10 @@ public MediaImageSharpConfiguration(IOptions<MediaOptions> mediaOptions)
         public void Configure(ImageSharpMiddlewareOptions options)
         {
             options.Configuration = Configuration.Default;
-            options.BrowserMaxAge = TimeSpan.FromDays(_mediaOptions.MaxBrowserCacheDays);
-            options.CacheMaxAge = TimeSpan.FromDays(_mediaOptions.MaxCacheDays);
+            //options.BrowserMaxAge = TimeSpan.FromDays(_mediaOptions.MaxBrowserCacheDays);
+            //options.CacheMaxAge = TimeSpan.FromDays(_mediaOptions.MaxCacheDays);
+            options.BrowserMaxAge = TimeSpan.FromSeconds(10);
```
Comment (sebastienros) [inline src/OrchardCore.Modules/OrchardCore.Media/Processing/MediaImageSharpConfiguration.cs:28]: Looks like debug values

File: src/OrchardCore.Cms.Web/appsettings.json [hunk 2]
```diff
@@ -64,15 +66,14 @@
...
-    //{
-    //  "ConnectionString": "", // Set to your Azure Storage account connection string.
-    //  "ContainerName": "somecontainer", // Set to the Azure Blob container name. Templatable, refer docs.
-    //  "BasePath": "some/base/path", // Optionally, set to a path to store media in a subdirectory inside your container. Templatable, refer docs.
-    //  "CreateContainer": true // Activates an event to create the container if it does not already exist.
-    //  "RemoveContainer": true // Whether the 'Container' is deleted if the tenant is removed, false by default.
-    //},
-    // See https://stackexchange.github.io/StackExchange.Redis/Configuration.html
+    "OrchardCore_Media_Azure": {
+      "ConnectionString": "UseDevelopmentStorage=true", // Set to your Azure Storage account connection string.
```
Comment (sebastienros) [inline src/OrchardCore.Cms.Web/appsettings.json:70]: Looks like debug values

File: src/OrchardCore.Cms.Web/appsettings.json [hunk 3]
```diff
@@ -48,7 +48,9 @@
     //  "UseTokenizedQueryString": true,
     //  "AllowedFileExtensions": [".jpg",".jpeg",".png",".gif",".ico",".svg",".webp",".pdf",".doc",".docx",".ppt",".pptx",".pps",".ppsx",".odt",".xls",".xlsx",".psd",".mp3",".m4a",".ogg",".wav",".mp4",".m4v",".mov",".wmv",".avi",".mpg",".ogv",".3gp"],
     //  "ContentSecurityPolicy": "default-src 'self'; style-src 'unsafe-inline'"
-    //}
+      "ResizedCacheMaxStale": "00:00:02:00", // The time before a staled item is removed from the resized media cache, if not provided there is no cleanup.
```
Comment (sebastienros) [inline src/OrchardCore.Cms.Web/appsettings.json:51]: Looks like debug values

File: src/OrchardCore.Modules/OrchardCore.Media/Processing/MediaImageSharpConfiguration.cs (hunk 1 above)
Comment (jtkech) [reply src/OrchardCore.Modules/OrchardCore.Media/Processing/MediaImageSharpConfiguration.cs:28]: Yes they are for testing

File: src/OrchardCore.Cms.Web/appsettings.json (hunk 2 above)
Comment (jtkech) [reply src/OrchardCore.Cms.Web/appsettings.json:70]: Yes they are for testing

File: src/OrchardCore.Cms.Web/appsettings.json (hunk 3 above)
Comment (jtkech) [reply src/OrchardCore.Cms.Web/appsettings.json:51]: Yes they are for testing

**Comment by jtkech** [comment]
I'm merging this one because nothing happens if the related config values are not provided.

**Pull request merged**